doc/collaborations.rst

```diff
@@ -1,14 +1,13 @@
-=======================
 Collabortation Diagrams
```

Comment:
It's not clear to me that it's useful to publish this document.

Reply:
Not sure either, and honestly, without more explanation and context I would not know what useful information to read.
I was thinking that putting all the articles on the light, even technical ones like this, could (maybe) make people want to improve it.

Still, I can unpublish it if you prefer.

Reply:
My major concern is that, with just a quick glance, I saw things that I'm pretty sure are now just flat-out wrong. It refers to a `register` method on arbitrary resource managers, but that was deprecated functionality and is no longer supported. There's also talk of a `discard` method that I don't think exists.

If you find value in publishing it, that's fine, but I would suggest adding a big disclaimer to the top:
```
.. caution:: 
    This document hasn't been reviewed since 2005 
    and is likely out of date.
```

Reply:
Ok I have put disclaimers on the old files.